smoke-tests/spec/modsec_default_spec.rb

```diff
@@ -0,0 +1,81 @@
+require "spec_helper"
+
+# The spec in this file is for testing ModSecurity with default "nginx"
+# ingress controller by using the ingress_class = "nginx" in their ingress annotation
+
+describe "Testing modsec" do
```

Comment:
Starting to think we should change the description to Testing modsec on ingress class: "nginx"